src/monodroid/monodroid.props

```diff
@@ -9,6 +9,7 @@
     <_HostCommonWinCFlags>$(_CommonCFlags) -DWINDOWS -DNTDDI_VERSION=NTDDI_VISTA -D_WIN32_WINNT=_WIN32_WINNT_VISTA -fomit-frame-pointer</_HostCommonWinCFlags>
     <_HostCommonWinLdFlags>-Wall -lstdc++ -lz -shared -fpic -ldl -lmman -pthread -lwsock32 -lole32 -luuid</_HostCommonWinLdFlags>
     <_UnixAdditionalSourceFiles>$(MonoSourceFullPath)\support\nl.c jni\debug.c jni\monodroid-networkinfo.c jni\xamarin_getifaddrs.c</_UnixAdditionalSourceFiles>
+    <_LinuxFlatPakBuild Condition="Exists('/app/')" >-DLINUX_FLATPAK</_LinuxFlatPakBuild>
```

Comment:
Considering that the system lib is `/app/lib/mono`, it would make sense to use that for detection of the targed IMO :)